xbmc/guilib/GUIInfoTypes.h

```diff
@@ -117,6 +118,10 @@ class CGUIInfoLabel
    */
   static std::string ReplaceAddonStrings(const std::string &label);
 
+  typedef boost::function<std::string(const std::string&)> StringReplacerFunc;
```

Comment:
Replace this with std::function now that we use c++11. We're moving away from boost so taking any new dependencies on it should be avoided.